applications/smart-distancing/eval/README.md

```diff
@@ -0,0 +1,52 @@
+# Smart Social Distancing - Performance Metrics for Object Detection
+
+This module provides easy-to-use functions which enables you to evaluate available exported results from detectors.
+
+### How to use
+**Step 1: Build Image**
+```
+cd neuralet/applications/smart-distancing/eval
+# Build Docker image
+docker build -f Dockerfile-eval -t detector-eval .
+```
+**Step 2: Create The Input Files**
+
+* Create a separate ground truth text file for each image in the directory `neuralet/applications/smart-distancing/eval/eval_files/groundtruths/`.
+* In these files each line should be in the format: `<class_name> <left> <top> <width> <height>`.
+* E.g. A sample of ground truth bounding boxes of an image:
+```
+bottle 6 234 39 128
+person 1 156 102 180
+person 36 111 162 305
+person 91 42 247 458
+```
+* Create a separate detection text file for each image in the directory `neuralet/applications/smart-distancing/eval/eval_files/detresults/`
+* The names of the detection files must match their correspond ground truth (e.g. "detections/2008_000182.txt" represents the detections of the ground truth: "groundtruths/2008_000182.txt").
```

Comment:
you used `detresults` earlier and used `detections` in here. it can be confusing.